src/components/Header/NavHeaderWithAlert.js

```diff
@@ -0,0 +1,99 @@
+import React, {
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+import { withStyles } from '@material-ui/styles';
+import axios from 'axios';
+import HeaderView from './HeaderView';
+import NavBarContainer from '../NavBar/NavBarContainer';
+
+const urlLink = 'https://cbiit.github.io/nci-softwaresolutions-elements/banners/government-shutdown.html';
```

Comment:
@rana22 this is url for analysis teams. Can we replace with one with **"https://cbiit.github.io/crdc-alert-elements/banners/government-shutdown.html"**